arduino/discovery/discovery.go

```diff
@@ -117,24 +122,30 @@ func (disc *PluggableDiscovery) jsonDecodeLoop(in io.Reader, outChan chan<- *dis
 	closeAndReportError := func(err error) {
 		disc.statusMutex.Lock()
 		disc.alive = false
+		if debug {
+			fmt.Println("ERR:", err)
```

Comment:
Can't we just use `logrus.debug`?

Reply:
I guess I'll just remove it...

Reply:
I very much prefer if we keep some debugging capabilities, I'd just prefer if we do it in a more structured way.